### PR TITLE
switch upstart role to using multilog

### PR DESCRIPTION
**Pull request by opadron**
Switches upstart role so that all generated services log their output through the `multilog` tool from the `daemon-tools` package instead of directly to disk.  The `multilog` tool handles rotating logs and limiting log size, so we should no longer be running into megabyte- and gigabyte-size logs.

A side-effect of this change is that markers for service startup and termination are no longer inserted into the log output.

### CODE REVIEW

**Comment by opadron** [comment]
Hey, @kotfic, could you QA this?  It should be a quick one and would be nice to have.

**Comment by kotfic** [comment]
Sorry for long delay on review of this - I am not super familiar with multilog/daemontools.  It looks like a primarily debian/ubuntu thing?  At the very least this should probably ensure that multilog is installed.  A default VM of ubuntu 14.04 doesn't come with daemontools installed.

**Comment by opadron** [comment]
multilog comes with the daemontools package.  I don't think it's specific to Debian.

**Comment by kotfic** [comment]
sure,  lets just make sure there is an apt call that insures daemontools is installed.

File: roles/upstart/tasks/main.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ---
 
+  - name: daemontools | install
+    apt: name=daemontools state=present update_cache=yes
```
Comment (opadron) [inline roles/upstart/tasks/main.yml:4]: The daemontools package should be installed here.

**Comment by opadron** [comment]
resolved via #61

**Pull request closed without merging**